examples/demo-stf/src/batch_builder.rs

```diff
@@ -89,6 +89,7 @@ where
 
             // Execute
             {
+                // TODO: Bug(!), because potential discrepancy with default stf or any other STF
```

Comment:
Let's handle this TODO (either by fixing it or adding an issue) I think this will be easier to resolve after we finish #361, so it's probably fine to just create an issue and link here.

Reply:
Agree, added link to the issue